utilities/create_site.sh

```diff
@@ -30,4 +30,17 @@ git add index.html
 echo "<html><head><script>window.location.replace('/google-cloud-java/${SITE_VERSION_BASE}/apidocs' + location.search)</script></head><body></body></html>" > apidocs/index.html
 git add apidocs/index.html
 
+cd ..
+mvn site:stage --quiet -Djava.util.logging.config.file=logging.properties -DtopSiteURL=http://googlecloudplatform.github.io/google-cloud-java/site/latest/
+cd tmp_gh-pages
+rm -rf latest
+mkdir latest
```

Comment:
I can answer I think - it's confusing to provide a link to GA library documentation (e.g. for pubsub, which is 1.x) but where the version in the link is the alpha version (0.x). Also, every single release we do, cloud docs have to be updated too, which is a big maintenance burden; having the /latest/ link is much easier to link to.